Create 0011-researcher-in-residence.md (#65)
MinBZK/ai-validation: docs/ADRs/0011-researcher-in-residence.md

In line with this reasoning, it is important to be open to researchers interested in the work of the AI validation team.
Allowing researchers to conduct research within the team contributes to transparency and enables external perspectives
and feedback to be incorporated into the team's work.

## Assumptions

- Having researchers in residence is a mechanism that facilitates transparency.
- Research enables external feedback and perspectives to be incorporated into the team's work.
- Having researchers in residence from other disciplines facilitates an interdisciplinary exchange of ideas in light of
  interdisciplinary issues.
- Research outcomes enable knowledge exchange between the scientific community and public organizations, in the
  Netherlands and abroad.

## Decision

We have decided to include a researcher in residence as a member of our team.

The researcher in residence takes the following form:

- They join and participate in refinement meetings and other meetings of relevance.
- They share their reflections and present their (interim) research findings.

> - They are provided access to the communication channel of the team (mattermost).

Reviewer: ```suggestion
- They are provided access to the communication channel of the team (Mattermost).
```

- They meet with a contact person bi-weekly to discuss questions and relevant progress and findings.
- The are independent as they are employed and financed by their respective university.
- The results of their research, and thus processed data, are published in an academic journal.

The following conditions apply to the researcher in residence.

- The research is able to access and analyze documents relevant to the research (ex. Notes/minutes taken). These
  documents are submitted to a member of the team for review prior to being processed.
- Any data collected and analyzed via interviews is done on the basis of informed consent.
- The data collected is not shared with other parties and is handled ethically by the researcher.

> - No information, aggregation, or summary of information from the communication channel of the team (mattermost) is

Reviewer: ```suggestion
- No information, aggregation, or summary of information from the communication channel of the team (Mattermost) is
```

  collected, processed, or analyzed by the researcher.

## Risks

> In light of aforementioned form and conditions above, we see no foreseeable significant risks.

Reviewer: ```suggestion
Risks around a potential chilling effect (team members not feeling free to express themselves) are mitigated by the conditions we impose. 
In light of aforementioned form and conditions above, we see no further significant risks. 
```

## Consequences

Including a researcher in residence makes it easier for them to conduct research within both the team and the wider
organization where the AI validation team operates. This benefits the quality of the research findings and the feedback
provided to the team and organization.

> ## More Information

Reviewer: ```suggestion
```
